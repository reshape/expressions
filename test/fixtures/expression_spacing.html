x {{foo}} x {{ foo }} x {{{ foo }}} x {{{foo}}} x
x{{foo}}x{{ foo }}x{{{ foo }}}x{{{foo}}}x
