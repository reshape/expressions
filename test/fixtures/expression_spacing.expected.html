x X x X x X x X x
xXxXxXxXx
